src/test/java/teammates/test/cases/ui/browsertests/AutomatedSessionRemindersTest.java

```diff
@@ -28,6 +29,20 @@ public static void classSetup() throws Exception {
         printTestClassHeader();
         testData = loadDataBundle("/AutomatedSessionRemindersTest.json");
         
+        /* 
+         * In this test, we set the email address of the accounts to be the same as the
+         * support email address. When running the test against a production server,
+         * email alerts will be sent to the specified support email address.
+         * The tester should manually check the email box after running the test suite. 
+         */
+        
+        testData.accounts.get("instructorWithEvals").email = Config.SUPPORT_EMAIL;
+        testData.instructors.get("AutSessRem.instructor").email = Config.SUPPORT_EMAIL;
+        testData.students.get("alice.tmms@AutSessRem.course").email = Config.SUPPORT_EMAIL;
+        testData.feedbackSessions.get("closingSession").creatorEmail = Config.SUPPORT_EMAIL;
+        testData.feedbackSessions.get("openingSession").creatorEmail = Config.SUPPORT_EMAIL;
+        testData.feedbackSessions.get("publishedSession").creatorEmail = Config.SUPPORT_EMAIL;
+        testData.feedbackQuestions.get("question").creatorEmail = Config.SUPPORT_EMAIL;
```

Comment:
Consider it done :+1:

Anyway, if this approach is better, we might as well move all other injections to the respective java files as well, there's not a lot of them. Advantages:

1. We can explain in the java files why the particular accounts are used.
2. We can remove the injection process to the json data files, which takes time and is applied to all regardless of presence of placeholders. This is again another optimization no matter how slight.

Of course, this is another issue.